SyslogNet.Client/Transport/SyslogLocalSender.cs

```diff
@@ -117,6 +118,16 @@ public void Send(IEnumerable<SyslogMessage> messages)
 			Send(messages, defaultSerializer);
 		}
 
+		public Task SendAsync(SyslogMessage message, ISyslogMessageSerializer serializer)
+		{
+			throw new NotImplementedException();

```

Comment:
Now that I read the latest comment in the commit, that SO article links to the [reasons](https://blogs.msdn.microsoft.com/pfxteam/2012/04/13/should-i-expose-synchronous-wrappers-for-asynchronous-methods/) why this is not a good idea.